src/main/resources/database/changelog/02-add-enable-daily-notifications.yaml

```diff
@@ -11,3 +11,4 @@ databaseChangeLog:
                   type: boolean
                   constraints:
                     nullable: false
+                  defaultValueBoolean: false
```

Comment:
As I've written - can this change presents in commit, where field was defined?

Reply:
No, that's too expensive to group changes by commits strictly. In order to fix something, we should commit fixes, not amend.